DevelopmentProcess.md

```diff
@@ -0,0 +1,37 @@
+# Development Process Guidelines
+
+These guidelines are indended to be used by [@morganey-lang] community
+members while working on [@morganey-lang]'s projects.
+
+They are completely optional, because [per rules](README.md#becoming-a-project-leader):
+
+> The Project Leaders are responsible for organizing the development process of the projects they lead.
+
+So, before contributing anything to any of the [@morganey-lang]'s projects, ask a corresponding Project Leader about development process details. See [Projects.md](Projects.md) for the full list of [@morganey-lang] projects and their corresponding Project Leaders.
```

Comment:
Probably add a line addressed to project leader, e.g. "if you're a project leader, consider following these practices with your development".